src/main/scala/ForComprehension.scala

```diff
@@ -0,0 +1,19 @@
+package com.knoldus
+
+object ForComprehension extends App{
+
+  private val ageOfVoters=List(20,17,23,55,19)
+  private val eligibility=for{
```

Comment:
for comprehension is better to use when there are many `.map` or .`flatmap`. Update this code to use multiple statements in `for comprehension` to utilise its power.